### DISCUSSION TITLE
[feature] Create a custom hook to determine if a list or content has reached your screen.

### DISCUSSION THREAD

**Discussion by ishanVaghasiya** [juliencrn/usehooks-ts#405] [Ideas]
custom React hook is designed to determine whether a dropdown list is positioned at the bottom of the screen or not.

> This can be useful in scenarios where you want to dynamically adjust the dropdown's position based on available screen space.

**Comment by AndreaPontrandolfo** [comment]
There is already https://floating-ui.com/. I think this is out-of-scope here.

**Reply by ishanVaghasiya** [reply]
Yes, but for this, the user needs to depend on https://floating-ui.com/ 
if the user has already set up https://usehooks-ts.com/ then no need to use another package.

**Comment by paras-goyani** [comment]
Yes, I need this feature.
In one of my projects, I independently handled the entire UI without using any UI packages and investing a significant amount of time in managing these kind of aspects.
This feature will undoubtedly be beneficial for other developers who also handle their own UI.
